test/NuGet.Core.Tests/NuGet.Packaging.Test/PackageBuilderTest.cs

```diff
@@ -2485,6 +2487,73 @@ var contentTypesReader
             }
         }
 
+        [Fact]
+        public void Icon_IconMaxFileSizeExceeded_ThrowsException()
+        {
+            TestIconPackaging(
+                new IconTestSourceDirectory("icon.jpg", "icon.jpg", PackageBuilder.MaxIconFileSize + 1),
```

Comment:
use named parameters for the arguments. It's hard to guess why the first two parameters have the same value mean reading only this test method.